include/SFML/Graphics/RenderTarget.hpp

```diff
@@ -525,6 +537,12 @@ class SFML_GRAPHICS_API RenderTarget
 /// OpenGL states are not messed up by calling the
 /// pushGLStates/popGLStates functions.
 ///
+/// While render targets are moveable, it is not valid to move them
+/// between threads. This will cause your program to crash. The
+/// problem boils down to OpenGL being limited with regard to how it
+/// works in multithreaded environments. Please ensure you only move
+/// render targets within the same thread.
```

Comment:
I am not sure but I naively imagine we could build abstractions that transfer information from one thread's OpenGL context to another, so maybe it would be more accurate to tell the problem is that moving an instance between threads is not supported by SFML rather than an fundamental OpenGL limitation. Can someone familiar with OpenGL confirm?